orangecontrib/text/widgets/ownyt.py

```diff
@@ -30,12 +30,16 @@ class OWNYT(OWWidget):
 
     outputs = [(Output.CORPUS, Corpus)]
     want_main_area = False
+    resizing_enabled = False
+
+    QT_DATE_FORMAT = 'yyyy-MM-dd'
+    PY_DATE_FORMAT = '%Y-%m-%d'
 
     # Settings.
     recent_queries = Setting([])
     recent_api_keys = Setting([])
-    year_from = Setting("")
-    year_to = Setting("")
+    date_from = Setting(datetime(2000, 1, 1).date().strftime(PY_DATE_FORMAT))
```

Comment:
This does not work for me. Even if I set this to:
```py
date_from = Setting(datetime(2015, 1, 1).date().strftime(PY_DATE_FORMAT))
```
and clear the settings, I always get the date `2000-01-01` in *both* date fields.
Also, I suggest setting `date_from` to a newer date, since we can only get 1000 documents. Maybe to the beginning of the current year?